packages/commonwealth/client/scripts/views/pages/discussions/HeaderWithFilters/HeaderWithFilters.tsx

```diff
@@ -141,7 +147,11 @@ export const HeaderWithFilters = ({
     <div className="HeaderWithFilters">
       <div className="header-row">
         <CWText type="h3" fontWeight="semiBold" className="header-text">
-          {isUndefined(topic) ? 'All Discussions' : topic}
+          {
+            isUndefined(topic)
+              ? ( onArchivePage ? 'Archived' : 'All Discussions')
```

Comment:
prettier: file is not formatted correctly

Reply:
Fixed!